### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by libern** [Shaninnik/PWSwitch#9]
![image](https://user-images.githubusercontent.com/1830640/51800306-3608cb80-2268-11e9-9937-75b9765e21b6.png)

Xcode 10.1
```
Type 'CAMediaTimingFillMode' (aka 'NSString') has no member 'forwards'
```

**Comment by libern** [comment]
Should change to 

```
kCAFillModeForwards
```

**Comment by Shaninnik** [comment]
No it is not. It was changed to `CAMediaTimingFillMode` in Swift 4.2. I forgot to update example project to Swift 4.2, fixed now f37a4c51a8bf42b892c89c31fc529d8e7f4dc457 PWSwitch project itself was building fine.

**Comment by libern** [comment]
Good! 👍
